src/styles.scss

```
@@ -26,7 +26,10 @@ $font-family-base: Arial;
 .hero-image {
   background-image: url(/public/me.jpg);
   /*background-color: #cccccc;*/
-  height: 600px;
+  min-height: 25rem;
+  min-width: 25rem;
+  width: 100%;
```

min-width is too much, if you want it to scale 100%
```suggestion
  width: 100%;
```